.NET/Microsoft.Recognizers.Definitions.Common/German/DateTimeDefinitions.cs

```diff
@@ -99,7 +100,7 @@ public static class DateTimeDefinitions
       public static readonly string DateExtractor7 = $@"({DayRegex}\s*[\.]\s*{MonthNumRegex}[\.])";
       public static readonly string DateExtractor8 = $@"(?<=\b(am)\s+){DayRegex}[/\\\.]{MonthNumRegex}[/\\\.]({DateYearRegex})?\b";
       public static readonly string DateExtractor9 = $@"\b({DayRegex}\s*/\s*{MonthNumRegex}((\s+|\s*,\s*){DateYearRegex})?)\b";
-      public static readonly string DateExtractor10 = $@"\b({RelativeRegex}\s*jahr(\s*im)?({MonthNumRegex}|sommer|winter|frühling|herbst)?)\b";

```

Comment:
Where are such cases supported now?

Reply:
@tellarin, in English a pattern like "next year in summer" is extracted as 2 separate daterange entities. In German, it is processed in the same way by DatePeriod but it is also still wrongly extracted as a single entity by DateExtractor (but not parsed). 
I will remove this pattern from German DateExtractor to conform German to English.
Should I also modify both languages to extract "next year in summer" as a single daterange entity?